### PR TITLE
Commit for migrate to null-safety

### PR DESCRIPTION
**Pull request by bhavin-concetto**
- Convert to Null Safety using Tool for #18 
 - Need to check

### CODE REVIEW

**Comment by bigflood** [comment]
@bhavin-concetto 
You should exclude .idea directory.

File: test/test_util.dart [hunk 1]
```diff
@@ -25,15 +25,15 @@ Stream<SampleFile> readSamples() async* {
 
   if (!await io.File(path).exists()) {
     print('downloading $path ..');
-    var res = await http.get(url);
+    var res = await http.get(url as Uri);
```
Comment (bigflood) [inline test/test_util.dart:28]: `url as Uri` -> `Uri.parse(url)`

**Comment by bhavin-concetto** [comment]
@bigflood I have updated the code but the one test is still not passing.

**Comment by bigflood** [comment]
@bhavin-concetto 

I updated `samples-exif.json` file.
You need to merge this commit -> https://github.com/bigflood/dartexif/commit/98f6988cd9a237d44622733b21d2642eb99fc99a

**Comment by bigflood** [comment]
@bhavin-concetto 
I would appreciate it if you could fix `unnecessary_null_comparison` warnings.

```
# dartanalyzer lib
Analyzing lib...
  hint • The operand can't be null, so the condition is always false. • lib/src/linereader.dart:42:13 • unnecessary_null_comparison
  hint • The operand can't be null, so the condition is always false. • lib/src/makernote_olympus.dart:32:11 • unnecessary_null_comparison
  hint • The operand can't be null, so the condition is always false. • lib/src/makernote_olympus.dart:36:11 • unnecessary_null_comparison
3 hints found.
```

**Comment by bhavin-concetto** [comment]
@bigflood  please check now.

**Comment by bigflood** [comment]
@bhavin-concetto 
Good!

**Pull request merged**